code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/filter/serviceevent/ServiceEventFilter.java

```diff
@@ -33,6 +32,10 @@
 public class ServiceEventFilter extends AbstractDefaultResourceManagerFilter {
 
     private static List<String> invalidStates = Arrays.asList(CommonStatesConstants.REMOVED, CommonStatesConstants.REMOVING);
+    private static List<String> upgradingStates = Arrays.asList(
+            ServiceConstants.STATE_UPGRADING,
```

Comment:
we should put all upgrade related states to the list, like `canceled-upgrade`, `upgraded`, `finishing-upgrade`